staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -193,20 +194,24 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				obj = scope.FieldManager.UpdateNoErrors(liveObj, obj, managerOrUserAgent(options.FieldManager, req.UserAgent()))
 				admit = fieldmanager.NewManagedFieldsValidatingAdmissionController(admit)
 			}
+			trace.Step("About to call mutatingAdmission")
 			if mutatingAdmission, ok := admit.(admission.MutationInterface); ok && mutatingAdmission.Handles(admission.Create) {
 				if err := mutatingAdmission.Admit(ctx, admissionAttributes, scope); err != nil {
 					return nil, err
 				}
 			}
+			trace.Step("mutatingAdmission.Admin finished")
 			// Dedup owner references again after mutating admission happens
 			dedupOwnerReferencesAndAddWarning(obj, req.Context(), true)
 			result, err := requestFunc()
+			trace.Step("Object stored in database", utiltrace.Field{"len", len(body)}, utiltrace.Field{"err", err})
```

Comment:
this is misleading because here we actually don't know if the object was stored - it could have ended with an error

Maybe write to database finished or sth?

Reply:
Done